docs/collections/_policies/json-format.md

```diff
@@ -1116,3 +1123,102 @@ JSON representation
     }
 ]
 ```
+
+## Representing a policy set with JSON
+
+### Overview
+
+Here is an example policy set containing a static policy and policy template.
+
+```cedar
+permit (
+    principal == User::"12UA45",
+    action == Action::"view",
+    resource in Folder::"abc"
+);
+
+permit (
+    principal == User::"12UA45",
```

Comment:
Nit -- change this one to a `forbid` just to make the example a little more expansive?

Reply:
Done